core/editor/config-providers/editor-v2-config-provider.php

```diff
@@ -21,6 +21,7 @@ class Editor_V2_Config_Provider implements Config_Provider_Interface {
 	];
 
 	const UTIL_PACKAGES = [
+		'documents',
```

Comment:
I think this is still an extension, it cannot live outside of the editor like `store` or `locations`